include/HelixSolver/Constants.h

```diff
@@ -8,34 +8,28 @@ static constexpr float PHI_BEGIN = -3.1416;
 static constexpr float PHI_END = 3.1416;
 static constexpr uint32_t MAX_Point_NUM = 100000;
 
+// Bool values for CDEBUG, used to surpress output
```

Comment:
Would it make more sense to have them defined in `Debug.h`

Reply:
I think it would make sense. I'll change that